crates/rolldown/tests/rollup/mutations-in-imports/artifacts.snap

```diff
@@ -0,0 +1,34 @@
+---
+source: crates/rolldown_testing/src/integration_test.rs
+snapshot_kind: text
+---
+# warnings
+
+## IMPORT_IS_UNDEFINED
+
+```text
+[IMPORT_IS_UNDEFINED] Warning: Import `value` will always be undefined because there is no matching export in 'foo.js'
+   ╭─[main.js:4:1]
+   │
+ 4 │ foo.value.a = 1;
+   │ ─────┬─────  
+   │      ╰─────── 
+───╯
+
+```
+# Assets
+
+## main.js
+
+```js
+
+//#region foo.js
+const foo = { a: 0 };
+
+//#endregion
+//#region main.js
+foo.a = 1;
+(void 0).a = 1;
+
```

Comment:
The esbuild repl result https://hyrious.me/esbuild-repl/?version=0.24.0&b=e%00entry.js%00import+*+as+file+from+%27.%2Ffile%27%0A%0Afile.y.a+%3D1%0Afile.b.a+%3D1&b=%00file.js%00export+const+b+%3D+%7B+a%3A+2+%7D&o=--bundle+--outdir%3Ddist+--external%3Aexternal+--format%3Desm